syntaxes/asterisk.tmLanguage

```diff
@@ -83,6 +83,35 @@
 				</dict>
 			</dict>
 
+			<!-- 'same' keywords continues an extension declaration with priorities and priority labels-->
+			<dict>
+				<key>match</key>
+				<string>(^same  =>?)(\s+(n|\d+)\b(\(([\w\d_-]+)\))?)?</string>
```

Comment:
Propose changing "same  =>" to "same\s+=>" to allow for variable number of space characters.

Reply:
Thanks for your comments.

In fact I hesitated between `same  =>` and `same\s+=>`.
I finally chose to propose the pattern with a fixed number of space because :
1. for the exten pattern match there was a fixed number of space (`exten =>` and not `exten\s+=>`)
1. and it was a way to 'force' the equal sign to be on the same level with exten or same:
```
; Good
exten = 1,1,NoOp
same  = n,Dial

; 'Bad'
exten = 1,1,NoOp
same = n,Dial
```

But enforcing this latter point is probably not the role of a syntax highlighter...
Therefore I propose to change both exten and same 
- to have an undertimined number of spaces between the keyword and the equal sign
- but to keep the keyword as the first word of the line though:
`(^exten =>?)` to `(^exten\s+=>?)`
and 
`(^same  =>?)` to `(^same\s+=>?)`

What do you think ?

Reply:
I agree we shouldn't force a particular indentation style. Changing both exten and same to allow for a variable number of spaces before the equals is a good fix. Let's roll with that.